app/controllers/declaration/ItemTypeController.scala

```diff
@@ -103,8 +103,11 @@ class ItemTypeController @Inject()(
     val itemTypeUpdated = updateCachedItemTypeAddition(itemId, itemTypeInput, itemTypeCache)
     ItemTypeValidator.validateOnAddition(itemTypeUpdated) match {
       case Valid =>
-        updateExportsCache(itemId, itemTypeUpdated).flatMap { _ =>
-          refreshPage(itemId, itemTypeInput, hasFiscalReferences)
+        updateExportsCache(itemId, itemTypeUpdated).map {
+          case Some(model) =>
+            refreshPage(itemId, itemTypeInput, model)
+          case None =>
+            InternalServerError("Declaration not updated")
```

Comment:
I dont think we should ever throw an `InternalServerError`. This could happen the TTL removes a users session which is normal behaviour. I would argue we should just redirect to the Start Page, or handle it in a way that doesn't create a bad user experience

Reply:
Could redirect to either `export-items` or `start` I guess depending on which you think is most suitable

Reply:
export-items would be better - if there are no declaration there should be second redirect